src/dharma.ts

```diff
@@ -16,6 +16,20 @@ import * as DharmaTypes from "./types";
 class Dharma {
     public static Types = DharmaTypes;
 
+    public static withBlockchainNode(blockchainNode: string) {
```

Comment:
i would write this signature as:

`public static initialize(host: string): Dharma`